### PR TITLE
Updated CI fix document

### PR DESCRIPTION
**Pull request by raych1**
Added guidelines for below items:
how to run model validation and semantic validation;
how to fix model validation and semantic validation.

### CODE REVIEW

File: documentation/ci-fix.md [hunk 1]
```diff
@@ -22,16 +22,23 @@ npm install; npm run prettier -- --write "specification/<service>/**/*.json"
...
-
-## How to fix model validation and semantic checks
+Run Model Validation locally:
+```
+npm install -g oav
+oav validate-example <swagger-spec-path>
+```
+Please see [readme](https://github.com/Azure/oav/blob/master/README.md) for how to install or run tool in details.
+Or you can run it in [OpenAPI Hub](https://portal.azure-devex-tools.com/tools/static-validation/static/errors/default)
+And refer to [guideline](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Semantic-and-Model-Violations-Reference.md) to check error code and how to fix in detail.
```
Comment (josefree) [inline documentation/ci-fix.md]: Suggest updates to "And refer to..."

Refer to [Semantic and Model Violations Reference](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Semantic-and-Model-Violations-Reference.md) for detailed description of validations and how-to-fix guidance.

File: documentation/ci-fix.md [hunk 2]
```diff
@@ -22,16 +22,23 @@ npm install; npm run prettier -- --write "specification/<service>/**/*.json"
...
-Here is the [guideline](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Semantic-and-Model-Violations-Reference.md) to understand the check failures and how to fix them.
+## Semantic Validation
+Run Semantic Validation locally:
+```
+npm install -g oav
+oav validate-spec <swagger-spec-path>
+```
+Please see [readme](https://github.com/Azure/oav/blob/master/README.md) for how to install or run tool in details.
+Or you can run it in [OpenAPI Hub](https://portal.azure-devex-tools.com/tools/static-validation/static/errors/default)
+And refer to [guideline](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Semantic-and-Model-Violations-Reference.md) to check error code and how to fix in detail.
```
Comment (josefree) [inline documentation/ci-fix.md]: The same as last comment.

File: documentation/ci-fix.md (hunk 1 above)
Comment (raych1) [reply documentation/ci-fix.md]: Updated.

**Review by josefree — APPROVED** [review]

**Comment by raych1** [comment]
@akning-ms , please review and merge it, thanks.

**Pull request merged**